ffmpeg: fix low quality in extracted frames

ioanungurean opened the pull request:
Added the `-qscale` parameter for the video `:v` with the quality `1` (1 - 33, 33 being the lowest quality)

Closes #1453

ioanungurean commented:
@ntarocco I update the PR and also installed an extension so that won't happen again. :)

ntarocco commented:
I have restarted the failing build on Travis.
No problem for the line length! ;)
Next time, if you want to keep your commit history clean, just `amend` your previous commit ;)

ioanungurean commented:
@ntarocco I actually started with amend but I reverted and added a new commit - I was thinking that you can keep track better if a do a new commit - not necessarily for this one :)  

It was working with `\` but for consistency I changed to what you suggested and added `()` because it was trowing some errors. I also added only one commit for this and reverted the other 2.
